tests/i18n/test_compilation.py

```diff
@@ -35,9 +35,13 @@ class PoFileTests(MessageCompilationTests):
     MO_FILE = 'locale/%s/LC_MESSAGES/django.mo' % LOCALE
 
     def test_bom_rejection(self):
-        with self.assertRaises(CommandError) as cm:
-            call_command('compilemessages', locale=[self.LOCALE], stdout=StringIO())
-        self.assertIn("file has a BOM (Byte Order Mark)", cm.exception.args[0])
+        err_buffer = StringIO()
```

Comment:
I wouldn't refactor the other test but `stderr` instead of `err_buffer` is more consistent with the majority of tests.